packages/nuxi/src/commands/dev.ts

```diff
@@ -69,7 +69,7 @@ export default defineNuxtCommand({
 
     // Watch for config changes
     // TODO: Watcher service, modules, and requireTree
-    const dLoad = debounce(debounce.promise(load), 250)
+    const dLoad = debounce(load, 250)
```

Comment:
Maybe use a default 100ms?
```suggestion
    const dLoad = debounce(load, 100)
```